client/js/nonprofits/donate/wizard.js

```diff
@@ -154,6 +154,28 @@ const postDedication = (dedication, donor, donation) => {
   }).load)
 }
 
+const titleInfo = state => {
+  if (state.params$().title_image_url) {
+    return [
+      h('img', {
+        props: {
+          src: state.params$().title_image_url
+        , alt: state.params$().title_image_alt || app.campaign.tagline || app.nonprofit.tagline || ''
```

Comment:
should we have a separate param here? should we instruct folks to use the taglines? should we include the designation && single_amount stuff from the existing line below (starting 171)

Reply:
I think we need a separate param for title_image_alt, there's going to be situations where the title image has nothing to do with the tagline of the campaign or nonprofit.

The question is what we should do if the they choose not to enter an alt. And I don't know.

Reply:
I think we'll go with this as-is in regards to the alt defaults. I don't know of the right way to handle this.